node_modules/oae-tenants/lib/rest.js

```diff
@@ -206,13 +206,7 @@ OAE.globalAdminRouter.on('post', '/api/tenantNetwork/:id/removeTenants', functio
  * @HttpResponse                    400             The tenant alias should not contain a space
  */
 OAE.globalAdminRouter.on('post', '/api/tenant/create', function(req, res) {
-    var opts = {};
-
-    // Apply the email domain if it is specified
-    if (_.isString(req.body.emailDomain)) {
-        opts.emailDomain = req.body.emailDomain;
-    }
-
+    var opts = _.oaeExtendDefined({}, _.pick(req.body, 'emailDomain', 'countryCode'));
```

Comment:
The API will throw an error if misused, however `alias` and `displayName`, for example, are not valid, so we have to pick these off.